en/guide/toolchain.md

```diff
@@ -0,0 +1,213 @@
+# Building C++ DroneCore Apps
```

Comment:
@julianoes OK, I know you approved the PR, but I've also updated this one to address the issues you raised - and if would be good if you could cast your eye over it. 

Main changes:
- The build section at the end corrected as you said
- Divided into system-wide installers and local installers. The system installer section is turnkey - just replace these fields. The local install adds the explanation - I know which I'd rather do!
- Unfortunately gitbook does not support source code line highlighting. I've requested that. 
- Reduced the Useful link section you were annoyed by.

I think this is quite clear.